fighthealthinsurance/models.py

```
@@ -1708,6 +1708,85 @@ def filter_to_allowed_attachments(cls, user):
+    Requires the model to have a `patient_user` ForeignKey to PatientUser.
+    """
+
+    @classmethod
+    def filter_to_allowed(cls, user: "User") -> models.QuerySet:
+        """Filter to records the user has permission to access."""
+        if not user.is_authenticated:
+            return cls.objects.none()  # type: ignore[attr-defined]
+        if user.is_superuser or user.is_staff:
+            return cls.objects.all()  # type: ignore[attr-defined]
+        try:
+            patient_user = PatientUser.objects.get(user=user)
```

Ruff (0.15.7)

[error] `PatientUser` may be undefined, or defined from star imports

(F405)